### PR TITLE
Release version v0.46.0

### LINKED ISSUE #759: Release 0.46.0
/assign

### PR DESCRIPTION
**Pull request by yuriolisa**
Signed-off-by: Yuri Sa <yurimsa@gmail.com>
Fixes #759

### CODE REVIEW

**Comment by yuriolisa** [comment]
Resolves #764

**Comment by yuriolisa** [comment]
@VineethReddy02 , @pavolloffay , @jpkrohling , could you please review it?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Changes by Version
 ==================
 
+0.46.0
+-------------------
+* Bumped OpenTelemetry Collector to v0.46.0
```
Comment (pavolloffay) [inline CHANGELOG.md:6]: Could you please add links to the collector core and contrib changelogs ?

File: README.md [hunk 2]
```diff
@@ -240,6 +240,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
+| v0.46.0                | v1.21 to v1.23       | 1.6.1                |
```
Comment (pavolloffay) [inline README.md:243]: This needs to be changed to 1.19 to 1.23

Comment (yuriolisa) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -240,26 +240,26 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
...
-| v0.39.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.38.0                | v1.20 to v1.22       | 1.6.1                |
-| v0.37.1                | v1.20 to v1.22       | v1.4.0 to v1.6.1     |
-| v0.37.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.36.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.35.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.34.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
-| v0.33.0                | v1.20 to v1.22       | v1.4.0 to v1.5.4     |
+| v0.46.0                | v1.19 to v1.23       | 1.6.1                |
+| v0.45.0                | v1.19 to v1.23       | 1.6.1                |
```
Comment (jpkrohling) [inline README.md:244]: We didn't test this with 1.19. While we are confident it works, we do not have positive confirmation that it does. I wouldn't change the past, so, just add a new 0.46.0 with a wider range of supported versions.

**Review by jpkrohling — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+* add autoscale option to enable support for Horizontal Pod Autoscaling ([#746](https://github.com/open-telemetry/opentelemetry-operator/pull/746), [@binjip978](https://github.com/binjip978))
+* chore(nodejs): bump auto-instrumentations ([#763](https://github.com/open-telemetry/opentelemetry-operator/pull/763), [@mat-rumian](https://github.com/mat-rumian))
+* Fix publishing main docker tag for operator image ([#758](https://github.com/open-telemetry/opentelemetry-operator/pull/758), [@pavolloffay](https://github.com/pavolloffay))
```
Comment (VineethReddy02) [inline CHANGELOG.md:9]: @yuriolisa we shouldn't be adding in CI-related changes or any changes that don't make sense to end users in `CHANGELOG` as this is more focused on changes that users should be aware of from the otel-operator side. 

cc @jpkrohling @pavolloffay

**Review by VineethReddy02 — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (yuriolisa) [reply CHANGELOG.md]: Thank you @VineethReddy02. I've just fixed it.

**Pull request merged**